### PR TITLE
Dev -> Main

### PR DESCRIPTION
**Pull request by Saverio976**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Norm of the code has been respected

* **In what state is your pull request?**
- [ ] Ready to merge / Waiting a reviwer to see my work.
- [ ] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [x] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of change does this PR introduce?** (You can choose multiple)
- [x] Bug fix
- [x] Feature request
- [x] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **What is the current behavior?** (link an issue based on the kind of change this pr introduce)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by kw4ntiq — APPROVED** [review]

File: docs/contributors.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contributors
+
+Thanks to this amazing contributors that made this project possible!
+
+- [@Saverio976](https://github.com/Saverio976)
+- [@TTENSHII](https://github.com/TTENSHII)
+- [@guillaumeAbel](https://github.com/guillaumeAbel)
+- [KitetsuK](https://github.com/KitetsuK)
```
Comment (KitetsuK) [inline docs/contributors.md]: J ai pas d'@ :/

Comment (kw4ntiq) [inline docs/contributors.md]:
```suggestion
- [@KitetsuK](https://github.com/KitetsuK)
```

Comment (Saverio976) [reply docs/contributors.md]: disoulé

**Review by romainpanno — APPROVED** [review]

**Review by romainpanno — CHANGES_REQUESTED** [review]
remove comments in formats' scripts .ps1 and .sh

**Comment by Saverio976** [comment]
Je corrige le build windows demain

File: scripts/compil.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/pwsh
+# Compil using cmake
+
+$ErrorActionPreference = 'Stop'
+
+cmake -S . -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+
+cmake --build build
```
Comment (kw4ntiq) [inline scripts/compil.ps1]:
```suggestion
cmake --build build
```
```suggestion
cmake --build build -- /m
```

**Comment by Saverio976** [comment]
@romainpanno c'est fait

**Review by guillaumeAbel — APPROVED** [review]

**Review by romainpanno — APPROVED** [review]

**Pull request merged**